integration/client/volumes/volumeclasses_test.go

```diff
@@ -451,3 +451,200 @@ func TestCreateProjectDefaultWithExistingClusterDefault(t *testing.T) {
 
 	t.Fatal("project default not found")
 }
+
+func TestProjectVolumeClassUpdateValidation(t *testing.T) {
```

Comment:
nit:

Could/should we combine `TestProjectVolumeClassUpdateValidation` and `TestClusterComputeClassUpdateValidation` so the same test cases are run on each type?

Reply:
No, we don't want the same test cases run on both types because there will already be a default ClusterVolumeClass when Acorn is installed. That means that the test won't be able to create a default ClusterVolumeClass. That is why they are separate.